package.json

```diff
@@ -83,18 +83,21 @@
     "@babel/preset-env": "^7.24.7",
     "@babel/preset-react": "^7.24.7",
     "@babel/preset-typescript": "^7.24.7",
+    "@chromatic-com/storybook": "^1",
```

Comment:
I would remove the introduction of the chromatic tool for another ticket and discuss it first internally

Reply:
Updated: https://github.com/aragon/ods/pull/215/commits/3e0fcecae25f6a86a9d53ec7585b348a36a239c8